resources/assets/coffee/react/_components/beatmap-icon.coffee

```diff
@@ -37,7 +37,9 @@ el = React.createElement
 
     showTitle = @props.showTitle && !@props.overrideVersion?
 
+    mode = if beatmap.convert then 'osu' else beatmap.mode
```

Comment:
![](http://puu.sh/q9iGx.pngl)

dunno, this seems sort of misleading, especially in this case where converts are shown at the very beginning